ui/apos/components/AposImportModal.vue

```diff
@@ -124,18 +124,18 @@ export default {
       formData.append('file', this.selectedFile);
 
       try {
-        await apos.http.post(`${this.moduleAction}/${this.action}`, {
-          busy: true,
+        apos.http.post(`${this.moduleAction}/${this.action}`, {
```

Comment:
await is missing here too

Reply:
replaces the try/catch with a chained catch call.